enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -1007,21 +1016,21 @@
     (mt/dataset sample-dataset
       ;; with-gtaps creates a new copy of the database. So make sure to do that before anything else. Gets really
       ;; confusing when `(mt/id)` and friends change value halfway through the test
-      (mt/with-gtaps {:gtaps      {:products
-                                   {:remappings {:category
-                                                 ["dimension"
-                                                  [:field (mt/id :products :category)
-                                                   nil]]}}}
-                      :attributes {"category" nil}}
+      (mt/with-gtaps {:gtaps {:products
+                              {:remappings {:category
+                                            ["dimension"
+                                             [:field (mt/id :products :category)
+                                              nil]]}}}}
         (mt/with-persistence-enabled [persist-models!]
           (mt/with-temp* [Card [model {:dataset       true
                                        :dataset_query (mt/mbql-query
                                                        products
                                                        ;; note does not include the field we have to filter on. No way
                                                        ;; to use the sandbox filter on the cached table
                                                        {:fields [$id $price]})}]]
-            ;; persist model
-            (persist-models!)
+            ;; persist model (as admin, so sandboxing is not applied to the persisted query)
+            (mt/with-test-user :crowberto
+              (persist-models!))
```

Comment:
This test was subtly wrong...the persisted model was actually a sandboxed query, but with an attribute value of `nil` which previously returned the entire results set. Now the persisted model is an actual non-sandboxed query.